web_console/dynamic-plugin/overview-dynamic-plugin.adoc

```diff
@@ -9,7 +9,9 @@ toc::[]
 [id="dynamic-plug-in-overview"]
 == About dynamic plugins
 
-Dynamic plugins are deployed as workloads on the cluster. They allow you to add custom pages and other extensions to your console user interface at runtime. The `ConsolePlugin` custom resource registers plugins with the console, and a cluster administrator enables plugins in the `console-operator` configuration.
+Dynamic plugins are loaded and interpreted from remote sources at runtime. The standard way to deliver and expose dynamic plugins to the console is through OLM Operators. The Operator will create a deployment on the platform with an HTTP server to host the plugin and expose it using a Kubernetes service.
```

Comment:
I might say "one way to deliver" instead of "standard." If customers are writing their own plugins, it's a big ask to create an operator to deploy and manage it, and it's not really needed.